builtin/game/falling.lua

```diff
@@ -22,7 +22,16 @@ core.register_entity(":__builtin:falling_node", {
 
 	set_node = function(self, node, meta)
 		self.node = node
-		self.meta = meta or {}
+		meta = meta or {}
+		if type(meta) ~= "table" then meta = meta:to_table() end
```

Comment:
I'd rather check the type of `meta.to_table`:
```Lua
if type(meta.to_table) == "function" then
	meta = meta:to_table()
end
```
It's also unlikely that the type of the inventory content changes. Either it's for all `userdata` or `table`. Please check which one that applies here (probably latter).